cg/cli/workflow/tomte/base.py

```diff
@@ -0,0 +1,19 @@
+"""CLI support to create config and/or start TOMTE."""
+
+import logging
+
+import click
+
+from cg.constants.constants import MetaApis
+from cg.meta.workflow.analysis import AnalysisAPI
+from cg.meta.workflow.tomte import TomteAnalysisAPI
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group(invoke_without_command=True)
+@click.pass_context
+def tomte(context: click.Context) -> None:
+    """gms/tomte analysis workflow."""
```

Comment:
```suggestion
    """Tomte analysis workflow."""
```

Reply:
I will leave this as gms/tomte - I think it's clearer and more consistent with other workflows were we specify nf-core/pipeline. This is not nf-core but gms/ so I think it's good to show it here to avoid confusion